cluster/configs/src/main/java/com/linkedin/openhouse/cluster/configs/ClusterProperties.java

```diff
@@ -54,7 +54,7 @@ public class ClusterProperties {
   @Value("${cluster.housetables.database.type:IN_MEMORY}")
   private String clusterHouseTablesDatabaseType;
 
-  @Value("${cluster.housetables.database.url:jdbc:h2:mem:htsdb;DB_CLOSE_DELAY=-1}")
+  @Value("${cluster.housetables.database.url:jdbc:h2:mem:htsdb;MODE=MYSQL;DB_CLOSE_DELAY=-1}")
```

Comment:
This is needed for `INSERT IGNORE` syntax which is not natively supported in H2's syntax.